.travis.yml

```diff
@@ -23,5 +23,15 @@ script:
   # Build Framework in Debug and Run Tests if specified
   - travis_retry xcodebuild -workspace "$WORKSPACE" -scheme "$SCHEME" -sdk "$SDK" -destination "$DESTINATION" -configuration Debug ONLY_ACTIVE_ARCH=NO ENABLE_TESTABILITY=YES test | xcpretty;
 
+  # Build iOS Example in Debug if specified
+  - if [ $BUILD_IOS_EXAMPLE == "YES" ]; then
+      travis_retry xcodebuild -workspace "$WORKSPACE" -scheme "$IOS_EXAMPLE_SCHEME" -sdk "$SDK" -destination "$DESTINATION" -configuration Debug ONLY_ACTIVE_ARCH=NO build | xcpretty;
+    fi
+
+  # Build tvOS Example in Debug if specified
+  - if [ $BUILD_TV_EXAMPLE == "YES" ]; then
+      travis_retry xcodebuild -workspace "$WORKSPACE" -scheme "$TVOS_EXAMPLE_SCHEME" -sdk "$SDK" -destination "$DESTINATION" -configuration Debug ONLY_ACTIVE_ARCH=NO build | xcpretty;
+    fi
```

Comment:
instead of these BUILD_IOS_EXAMPLE and BUILD_TVOS_EXAMPLE flags, could each matrix entry just define an `EXAMPLE_SCHEME` variable? That may simplify this script (and it may still check if `EXAMPLE_SCHEME` is defined if we don't want to do it for all matrix entries)